.github/workflows/multi-build.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     steps:
       - id: check
         run: |
-          if [[ '${{ github.event_name }}' == 'schedule' ]] || [[ '${{ github.event_name }}' == 'pull_request_review' ]] || [[ '${{ github.event.label.name }}' == 'multi-build' ]]; then
+          if [[ '${{ github.event_name }}' == 'schedule' ]] || [[ '${{ github.event_name }}' == 'pull_request_review' ]] || [[ '${{ github.event.label.name }}' == 'multi-build' ]] && (!endsWith(github.head_ref, 'docs') && !endsWith(github.head_ref, 'noci')); then
```

Comment:
`head_ref` might not be defined, maybe we will need a fallback as we have in the benchmark flow:
`${{ github.head_ref || github.ref_name }}`